resources/assets/components/CampaignInbox/index.js

```diff
@@ -72,6 +72,10 @@ class CampaignInbox extends React.Component {
   }
 
   updateTag(postId, tag) {
+    if (tag === 'Hide In Gallery 👻') {
```

Comment:
I wonder if it would be better to handle this logic in the Tags component, so that it sends the correct value to this function so you don't have to do the check here. Might help with keeping things less tangled.